doc/source/contrib/language_ref/property_ref/main_properties.rst

```diff
@@ -0,0 +1,237 @@
+checks
+------
+
+A dictionary of labelled checks each of which is a grouping of properties (see
+Supported Properties). Eack check is executed independently and produces a
+boolean *result* of True or False.
+
+Checks are normally implemented in conjunction with :ref:`conclusions`
+as part of :ref:`scenarios`.
+
+format:
+
+.. code-block:: yaml
+
+    checks:
+      check1:
+        <property1>
+        <property2>
+      check2:
+        <property3>
+        <property4>
+      ...
+
+    CACHE_KEYS
+      search
+      requires
+
+usage:
+
+.. code-block:: python
+
+    <checkname>.result
+
+Settings
+^^^^^^^^
+
+  * :ref:`search`
+  * :ref:`requires`
+  * :ref:`input`
+
+
+conclusions
+-----------
+
+This indicates that everything beneath is a set of one or more conclusions to
+be used by `scenarios <../hotsos/core/ycheck/scenarios.py>`_. The contents of
+this override are defined as a dictionary of conclusions labelled with
+meaningful names.
+
+A conclusion is defined as a function on the outcome of a set of checks along
+with the consequent behaviour should the conclusion match. This is defined as
+an issue type and message that will be raised. If multiple conclusions are
+defined, they are given a :ref:`priority` such that the highest one to
+match is the one that is executed. See :ref:`scenarios` section for more
+info and examples.
+
+The message can optionally use format fields which, if used, require
+format-dict to be provided with required key/value pairs. The values must be
+an importable attribute, property or method.
+
+format:
+
+.. code-block:: yaml
+
+    conclusions:
+      <name>:
+        priority: <int>
+        decision:
+          and|or: [check1, ...]
+        raises:
+          type: <import-path>
+          message: <formattable string>
+          format-dict:
+            <key>: <value>
+
+usage:
+
+.. code-block:: python
+
+    <conclusionname>.reached
+    <conclusionname>.priority
+    <conclusionname>.issue_message
+    <conclusionname>.issue_type
+
+Settings
+^^^^^^^^
+
+decision
+""""""""
+
+This property is typically used in :ref:`conclusions`.
+CHECKS refers to a set of one or more :ref:`checks` names organised as a
+:ref:`LogicalCollection` to make decision on the outcome of more
```

Comment:
s/to make a decision on the outcome of one or more

Reply:
ack.